pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusCommunityDepAnalyzer.java

```diff
@@ -314,11 +314,16 @@ private boolean hasQuarkusExtensionMetadata(Path path) {
     private Set<String> gatherProblematicDeps() {
         Set<String> problemmaticDeps = new TreeSet<>();
         problemmaticDeps.addAll(checkBomContents(".*/io/quarkus/quarkus-bom/.*\\.pom"));
-        problemmaticDeps.addAll(checkBomContents(".*/com/redhat/quarkus/quarkus-product-bom/.*\\.pom"));
+        if (Boolean.TRUE.equals(getAddOnConfiguration().get("coreOnly"))) {
```

Comment:
I think that we should reuse the check from https://github.com/project-ncl/bacon/pull/554/files#diff-9b88c7d8548b03f12f77aae69dd7ec253e5b47dc2b53cd73f1bda38d1259765bL269

WDYT @RotBolt ?

Reply:
I have changed it. Please see it.